### DISCUSSION TITLE
Clone repository mac

### DISCUSSION THREAD

**Discussion by mariamor99** [TheAlbertDev/MASB#42] [Q&A] [answered]
Hi Albert! 
I'm trying to clone the repository from mac but when I get to the _Git Bash here_ step, I don't have the option even though I've downloaded Git.
I don't know if it's a mac problem.
Thank you so much

**Comment by Albert-Alvarez** [comment, marked as answer]
Macs offer a terminal with which to operate Git directly. To open a terminal in a given folder, right click in the desired folder and go to "Services > New Terminal at Folder". This will open a terminal in that folder from which you can run Git commands directly (as long as you have already installed it).
